scripts/start-eigenda-proxy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
+
+if [ ! -e /data/verified ]; then
+	wget https://srs-mainnet.s3.amazonaws.com/kzg/g1.point --output-document=/data/g1.point
+	wget https://srs-mainnet.s3.amazonaws.com/kzg/g2.point.powerOf2 --output-document=/data/g2.point.powerOf2
+	wget https://raw.githubusercontent.com/Layr-Labs/eigenda-operator-setup/master/resources/srssha256sums.txt --output-document=/data/srssha256sums.txt
```

Comment:
Where do these come from?

nitpick: the indentation is not consistent.

Reply:
Added the reference in the script and updated indents.